backend/controllers/users.py

```diff
@@ -0,0 +1,85 @@
+from models.user import *
```

Comment:
import according to convention and avoid using `*` its a bad practice read about it online